README.md

```diff
@@ -1,8 +1,13 @@
-# edx-certificates
+# Archive Notice
+
+**This repository is in the process of being Archived.**
+Please see the DEPR ticket for more details: https://openedx.atlassian.net/browse/DEPR-160
+
 
```

Comment:
Note: All changes other than this archive notice were from my editor cleaning up spaces.